packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -300,6 +300,12 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
   /* @conditional-compile-remove(pinned-participants) */
   const [pinnedParticipantsState, setPinnedParticipantsState] = React.useState<string[]>([]);
   /* @conditional-compile-remove(pinned-participants) */
+  props.pinnedParticipants?.forEach((pinParticipant) => {
```

Comment:
This should be in a useEffect block to it only executes when the pinnedParticipants or the remoteParticipants changes

Reply:
done